### PR TITLE
add helm to k8s master for vault and fixes [#1729]

### PR DESCRIPTION
**Pull request by ar3ndt**

### CODE REVIEW

**Review by erzetpe — APPROVED** [review]

File: core/src/epicli/data/common/ansible/playbooks/roles/vault/tasks/main.yml [hunk 1]
```diff
@@ -34,9 +34,11 @@
     dest: /tmp/vault_helm_chart_values.yaml
   when: vault_helm_chart_values_bool
 
-- name: Install and configure Hashicorp Vault if enabled
+- name: Install and configure Hashicorp Vault if enabled and no cloud service is used
   when:
     - specification.vault_enabled
+    - k8s_as_cloud_service is not defined
```
Comment (sk4zuzu) [inline core/src/epicli/data/common/ansible/playbooks/roles/vault/tasks/main.yml:40]: I think this line has no effect, we can safely assume this is defined always (it's merged with defaults), no? :thinking: 
```suggestion
```

File: core/src/epicli/data/common/ansible/playbooks/roles/vault/tasks/main.yml [hunk 2]
```diff
@@ -34,9 +34,11 @@
     dest: /tmp/vault_helm_chart_values.yaml
   when: vault_helm_chart_values_bool
 
-- name: Install and configure Hashicorp Vault if enabled
+- name: Install and configure Hashicorp Vault if enabled and no cloud service is used
```
Comment (sk4zuzu) [inline core/src/epicli/data/common/ansible/playbooks/roles/vault/tasks/main.yml:37]:
```suggestion
- name: Install and configure Hashicorp Vault if enabled and no cloud k8s service is used
```

File: core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/tasks/main.yml [hunk 3]
```diff
@@ -103,3 +103,8 @@
   lineinfile:
     path: "/home/{{ admin_user.name }}/.bashrc"
     line: source <(kubectl completion bash)
+
+- name: Include Helm binary installation
```
Comment (sk4zuzu) [inline core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/tasks/main.yml:107]:
```suggestion
- name: Install Helm binary file
```

**Comment by rpudlowski93** [comment]
LGTM, tested

**Review by rpudlowski93 — APPROVED** [review]

File: core/src/epicli/data/common/ansible/playbooks/roles/vault/tasks/main.yml (hunk 1 above)
Comment (sk4zuzu) [reply core/src/epicli/data/common/ansible/playbooks/roles/vault/tasks/main.yml:40]: Also it could mask some future issues with upgrades... :thinking:

**Review by sk4zuzu — CHANGES_REQUESTED** [review]
:thinking:

**Pull request merged**